### PR TITLE
fix: polish translations

### PR DESCRIPTION
**Pull request by karniv00l**
## Description

Found 3 Polish words that needed fixing

- [x] I have read and understand the [CONTRIBUTING.md](https://github.com/payloadcms/payload/blob/main/CONTRIBUTING.md) document in this repository.

## Type of change

- [ ] Chore (non-breaking change which does not add functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Change to the [templates](../templates/) directory (does not affect core functionality)
- [ ] Change to the [examples](../examples/) directory (does not affect core functionality)
- [ ] This change requires a documentation update

## Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Existing test suite passes locally with my changes
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/payload/src/translations/pl.json [hunk 1]
```diff
@@ -180,7 +180,7 @@
     "deselectAllRows": "Odznacz wszystkie wiersze",
     "duplicate": "Zduplikuj",
     "duplicateWithoutSaving": "Zduplikuj bez zapisywania zmian",
-    "edit": "Edytować",
+    "edit": "Edytuj",
```
Comment (karniv00l) [inline packages/payload/src/translations/pl.json:183]: Edytować (to edit) => Edytuj (edit)

File: packages/payload/src/translations/pl.json [hunk 2]
```diff
@@ -339,7 +339,7 @@
     "noRowsFound": "Nie znaleziono {{label}}",
     "preview": "Podgląd",
     "problemRestoringVersion": "Wystąpił problem podczas przywracania tej wersji",
-    "publish": "Publikować",
+    "publish": "Publikuj",
```
Comment (karniv00l) [inline packages/payload/src/translations/pl.json:342]: same as edit

File: packages/payload/src/translations/pl.json [hunk 3]
```diff
@@ -274,7 +274,7 @@
     "near": "blisko"
   },
   "upload": {
-    "crop": "Uprawa",
+    "crop": "Przytnij",
```
Comment (karniv00l) [inline packages/payload/src/translations/pl.json:277]: That's yield/harvest

**Comment by jacobsfletch** [comment]
Thank you @karniv00l!

**Pull request merged**